kaldb/src/main/java/com/slack/kaldb/logstore/search/KaldbDistributedQueryService.java

```diff
@@ -80,6 +96,17 @@ public KaldbDistributedQueryService(
     searchMetadataTotalChangeCounter = meterRegistry.counter(SEARCH_METADATA_TOTAL_CHANGE_COUNTER);
     this.searchMetadataStore.addListener(this::updateStubs);
 
+    this.distributedQueryTotalNodes =
```

Comment:
Also, do we need this level of granularity in the data? I think it may be simpler if we can just count total number of requests, timed out requests and partial requests.

If we need it, I am curious why these values are guages? If we run multiple queries, a guage would only record the results of the last query. I think a counter that is incremented by the number of failed and successful nodes is better.

Reply:
Swapped these out for counters, and removed the rate gauge entirely since that can now be calculated from the counters.